environment/lib/near2eth-relay.js

```diff
@@ -186,7 +186,20 @@ class Near2EthRelay {
 
             // Get new light client block.
             // @ts-ignore
-            const lightClientBlock = await near.connection.provider.sendJsonRpc('next_light_client_block', [clientBlockHash]);
+            let lightClientBlock;
+            for (let i = 0; i <= MAX_WEB3_RETRIES; i++) {
+                if (i === MAX_WEB3_RETRIES) {
+                    console.error(`Failed ${MAX_WEB3_RETRIES} times`);
```

Comment:
Simple retry can be retried with robust.js `retry(async() => { await fn(); await sleep(1000)}, RETRIES)` or `backoff`